charts/ingress-nginx/templates/controller-deployment.yaml

```
@@ -54,7 +54,7 @@ spec:
       imagePullSecrets: {{ toYaml .Values.imagePullSecrets | nindent 8 }}
     {{- end }}
     {{- if .Values.controller.priorityClassName }}
-      priorityClassName: {{ .Values.controller.priorityClassName }}
+      priorityClassName: "{{ .Values.controller.priorityClassName }}"
```

Let's leverage the built-in helm function `quote` for this

```suggestion
      priorityClassName: {{ .Values.controller.priorityClassName | quote }}
```